kubernetes/samples/charts/traefik/README.md

```diff
@@ -4,22 +4,22 @@ This sample demonstrates how to install the Traefik ingress controller to provid
 load balancing for WebLogic clusters.
 
 ## Install the Traefik operator with a Helm chart
-This document is based on Traefik version 2.x with helm chart located at https://github.com/containous/traefik-helm-chart.
-For more information about Traefik refer to https://docs.traefik.io/
+This document is based on Traefik version 2.x with Helm chart at https://github.com/containous/traefik-helm-chart.
+For more information about Traefik, see https://docs.traefik.io/
 
 To install the Traefik operator in the `traefik` namespace with default settings:
 ```
 $ helm repo add traefik https://containous.github.io/traefik-helm-chart
 $ helm repo update
 $ helm install traefik-operator traefik/traefik --namespace traefik
 ```
-Or, with a given `values.yaml` Refer to https://github.com/containous/traefik-helm-chart/blob/master/traefik/values.yaml for more detail.
+Or, with a given `values.yaml` For more detailed information, see https://github.com/containous/traefik-helm-chart/blob/master/traefik/values.yaml for more detail.
 ```
 $ helm install traefik-operator traefik/traefik --namespace traefik --values values.yaml
 ```
 
 ## Update the Traefik operator
-After the Traefik operator is installed and running, if you want to change some configurations of the operator, use `helm upgrade` to achieve this.
+After the Traefik operator is installed and running, if you want to change some operator configurations, use `helm upgrade` to achieve this.
```

Comment:
use `helm upgrade` to achieve this. -> use `helm upgrade`:

Reply:
sample does not need upgrade. the default values.yaml is sufficient enough to run the sample

Reply:
Then you can remove these sentences.

Reply:
removed